src/referencemodel.cpp

```diff
@@ -7,14 +7,6 @@
 
 #include "referencemodel.h"
 
-enum Roles : int {
```

Comment:
Ist das notwendig? Warum sind die enums in die Header-Datei verschoben worden?

Reply:
Überbleibsel von einem kaputten Implementierungsansatz, ich habe es zurückverschoben